### PR TITLE
[Kubectl] Update RunCreate to follow more conventions

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:
Make RunCreate with options like other commands, and abstract raw() into separated func.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by niuzhenguo** [comment]
cc @dixudx @mengqiy

File: pkg/kubectl/cmd/create.go [hunk 1]
```diff
@@ -38,6 +38,8 @@ import (
 )
 
 type CreateOptions struct {
+	Out, ErrOut io.Writer
```
Comment (dixudx) [inline pkg/kubectl/cmd/create.go]: Better split to 2 lines?

**Review by dixudx — COMMENTED** [review]
Looks good.
But I'm not convinced that we really need to make this change.

**Comment by niuzhenguo** [comment]
thanks @dixudx , I think this makes RunCreate follow more conventions like other cmds and abstract RAW cmd more cleanly although it doesn't fix any issue.

**Pull request merged**